zappa/core.py

```diff
@@ -315,13 +316,18 @@ def __init__(
         else:
             self.manylinux_suffix_start = "cp39"
 
+        self.architecture = architecture
```

Comment:
Since arm64 architecture is not supported on python3.7 runtime,  should we validate the runtime and architecture here?